examples/hpc-enterprise-slurm.yaml

```diff
@@ -0,0 +1,231 @@
+  # for a list of valid family options with Slurm
+  family: schedmd-v5-slurm-22-05-9-hpc-centos-7
+  project: projects/schedmd-slurm-public/global/images/family
+  # Set to true for active cluster reconfiguration.
+  # Note that setting this option requires additional dependencies to be installed locally.
+  # https://github.com/GoogleCloudPlatform/hpc-toolkit/tree/main/community/modules/scheduler/schedmd-slurm-gcp-v5-controller#description
+  enable_reconfigure: true
+  # When set, active compute nodes will be cleaned up on destroy.
+  # Note that setting this option requires additional dependencies to be installed locally.
+  enable_cleanup_compute: true
+
+# Documentation for each of the modules used below can be found at
+# https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/main/modules/README.md
+
+deployment_groups:
+- group: primary
+  modules:
+  # Source is an embedded module, denoted by "modules/*" without ./, ../, /
+  # as a prefix. To refer to a local or community module, prefix with ./, ../ or /
+  # Example - ./modules/network/vpc
+  - id: network1
+    source: modules/network/pre-existing-vpc
+
+  - id: homefs
+    source: modules/file-system/filestore
+    use: [network1]
+    settings:
+      filestore_tier: BASIC_SSD
+      size_gb: 2560 # smallest size for BASIC_SSD
+      local_mount: /home
+
+  - id: projectsfs
+    source: modules/file-system/filestore
+    use: [network1]
+    settings:
+      filestore_tier: HIGH_SCALE_SSD
+      size_gb: 10240 # smallest size for HIGH_SCALE_SSD
+      local_mount: /projects
+
+  # This file system has an associated license cost.
+  # https://console.developers.google.com/marketplace/product/ddnstorage/exascaler-cloud
+  - id: scratchfs
+    source: community/modules/file-system/DDN-EXAScaler
+    use: [network1]
+    settings:
+      local_mount: /scratch
+
+  - id: n2_node_group
+    source: community/modules/compute/schedmd-slurm-gcp-v5-node-group
+    settings:
+      node_count_dynamic_max: 4
+      machine_type: n2-standard-2
+      instance_image:
+        family: $(vars.family)
+        project: $(vars.project)
+
+  - id: n2_partition
+    source: community/modules/compute/schedmd-slurm-gcp-v5-partition
+    use: [n2_node_group, network1, homefs, projectsfs, scratchfs]
+    settings:
+      partition_name: n2
+      exclusive: false # allows nodes to stay up after jobs are done
+      enable_placement: false # the default is: true
+      is_default: true
+
+  - id: c2_node_group
+    source: community/modules/compute/schedmd-slurm-gcp-v5-node-group
+    settings:
+      node_count_dynamic_max: 20
+      machine_type: c2-standard-60 # this is the default
+      instance_image:
+        family: $(vars.family)
+        project: $(vars.project)
+      bandwidth_tier: tier_1_enabled
+      disk_type: pd-ssd
+      disk_size_gb: 100
+      # the following two are true by default
+      exclusive: true # this must be true if enable_placement is true
+      enable_placement: true
```

Comment:
These got moved to the node group instead of the partition. Caught by make tests.